shared/src/util/Numbers.ts

```diff
@@ -177,25 +177,34 @@ export function formatTokenAmountCompact(amount: number, length = 4): string {
   amount = Math.abs(amount);
 
   if (amount > 1e6) {
-    return sign + amount.toLocaleString('en-US', {
-      style: 'decimal',
-      notation: 'compact',
-      compactDisplay: 'short',
-      maximumSignificantDigits: length,
-      minimumSignificantDigits: 2,
-    });
+    return (
+      sign +
+      amount.toLocaleString('en-US', {
+        style: 'decimal',
+        notation: 'compact',
+        compactDisplay: 'short',
+        maximumSignificantDigits: length,
+        minimumSignificantDigits: 2,
+      })
+    );
```

Comment:
Something up with your linter? Seems to be going back and forth on a few things like this.

Reply:
It seems to like it like this now, will keep an eye on it.